tests/test_dotpersistence.py

```diff
@@ -84,16 +83,14 @@ def test_load_data(self):
         s3_path2 = 's3://test-bucket/123/dot-scrapy/testspider/'
         self.assertEqual(self.instance._s3path, s3_path2)
         mocked_call.assert_called_with(
-            ['s3cmd', 'sync', '--no-preserve', s3_path2, '/tmp/.scrapy'])
+            ['aws', 's3', 'sync', s3_path2, '/tmp/.scrapy'])
 
     def test_store_data(self):
         mocked_call = mock.Mock()
         self.instance._call = mocked_call
         self.instance._store_data()
         mocked_call.assert_called_with(
-            ['s3cmd', 'sync', '--no-preserve',
-             '--multipart-chunk-size-mb=5120',
-             '--delete-removed', '/tmp/.scrapy',
+            ['aws', 's3', 'sync', '--delete', '/tmp/.scrapy',
```

Comment:
I can't reproduce that issue with multipart uploads for `awscli`(https://github.com/scrapy-plugins/scrapy-dotpersistence/pull/3), works fine with awscli.